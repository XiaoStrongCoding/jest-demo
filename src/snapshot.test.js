/**
 * toMatchSnapshot 配置快照
 * 在项目根目录下生成一个__snapshots__文件夹，下面有一个snapshot.test.js.snap快照文件
 * npm test snapshot.test -- -u 更新
 */

import {getConfig} from './snapshot'

// test('测试snapshot', () => {
//     expect(getConfig()).toEqual({
//         server: 'https://demo.com',
//         port: '8080'
//     })
// });

test('测试toMatchSnapshot', () => {
    expect(getConfig()).toMatchSnapshot()
});