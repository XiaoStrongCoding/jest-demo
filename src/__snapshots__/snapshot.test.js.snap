// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试toMatchSnapshot 1`] = `
Object {
  "port": "8089",
  "server": "https://demo.com",
}
`;
